forage-android/build.gradle

```diff
@@ -121,6 +121,7 @@ dependencies {
     implementation "javax.ws.rs:javax.ws.rs-api:2.1@jar"
 
     implementation "com.datadoghq:dd-sdk-android:1.19.2"
+    testImplementation 'androidx.test:core-ktx:1.5.0'
```

Comment:
Needed to add this in order to use [Robolectric](https://robolectric.org/) to create JVM unit tests for the `FormatPanTextWatcher`.